test/funnels/automation/StableSwapper.t.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.16;
+
+import "dss-test/DssTest.sol";
+import { Swapper, GemLike } from "src/funnels/Swapper.sol";
+import { StableSwapper } from "src/funnels/automation/StableSwapper.sol";
+import { UniV3SwapperCallee } from "src/funnels/callees/UniV3SwapperCallee.sol";
+import { AllocatorRoles } from "src/AllocatorRoles.sol";
+import { AllocatorBuffer } from "src/AllocatorBuffer.sol";
+import { TestUtils } from "test/utils/TestUtils.sol";
+
+contract StableSwapperTest is DssTest, TestUtils {
+    event Swap (address indexed sender, address indexed src, address indexed dst, uint256 amt, uint256 out);
```

Comment:
```suggestion
    event Swap(address indexed sender, address indexed src, address indexed dst, uint256 amt, uint256 out);
```

Reply:
Done in https://github.com/makerdao/dss-allocator/pull/15